src/test/resources/features/complaints/ComplaintsTriage.feature

```diff
@@ -202,6 +225,18 @@ Feature: Complaints Triage
     And I select the "No Further Consideration" action for an IEDET case at the Triage stage
     Then the case should be closed
 
+  @ComplaintsRegression
+  Scenario: User can hard close a BF case at the Triage stage
+    Given I am logged into "CS" as user "BF_USER"
+    When I create a "BF" case and move it to the "Triage" stage
+    And I load and claim the current case
```

Comment:
Please use 'I get a "BF" case at the "Triage" stage'. Then you wont require the second step about loading and claiming the case.